content-repo/extra-docs/articles/Email-Communication-Readme.md

```diff
@@ -0,0 +1,195 @@
+
+The pre-process script searches for the 8-digit number in the subject of the email to link to the incident. 
+
+If there is no 8-digit number in the subject of the email or if the pre-process rule is unable to locate the 8-digit number, a new incident is created and a random 8-digit number is generated for the incident. 
+
+If there is a 8-digit number in the subject of the email, and the pre-process rule is able to locate the incident associated with the 8-digit number, the email is added to the existing email thread and not as a separate unlinked email. 
+
+You can view the email thread in the War Room or in the incident layout. You can also reply to the email thread in the incident layout. 
+
+
+ 
+
+
+## In This Pack
+The Email Communication content pack includes several content items.
+
+### Automations
+There are 3 Automations in this pack.
+* [DisplayEmailHtml](https://xsoar.pan.dev/docs/reference/scripts/display-email-html): Displays the original email in HTML format in the incident layout.
+
+* [PreprocessEmail](https://xsoar.pan.dev/docs/reference/scripts/preprocess-email): Pre-process script for Email Communication layout. This script checks if the incoming email contains an 8-digit number to link the mail to an existing incident, and tags the email as "email-thread". 
+
+* [SendEmailReply](https://xsoar.pan.dev/docs/reference/scripts/send-email-reply): Sends the email reply. 
+
+### Classifiers
+There are 4 Classifiers in this pack. When you configure an instance of the Gmail integration or EWS V2 integration, use the following classifiers.
+
+* **EWS - Classifier - Email Communication**:  Classifies EWS email messages.
+* **EWS - Incoming Mapper - Email Communication**:  Maps incoming EWS email message fields. 
+* **Gmail - Classifier - Email Communication**:   Classifies Gmail email messages. 
+* **Gmail - Incoming Mapper - Email Communication**: Maps incoming Gmail email message fields.
+
+### Incident Fields
+There is 1 incident field - **Add CC To Email**. 
+
+### Incident Types
+There is 1 incident type - **Email Communication**.
+
+### Layout
+There is 1 layout - **Email Communication** 
+
+There are 3 interactive sections in which you can specify 1 or more email addresses to add as a CC to the email, create the body of the email, and add attachments. The remaining 3 sections are for viewing the original email and all the email communications and attachments associated with the incident.
+
+
+
+![Layout](https://raw.githubusercontent.com/demisto/content/84e7bc89c8757544804540e6711d4b9aba210ec1/Packs/EmailCommunication/doc_files/Email_Communication_layout.png)
+
+You can use the layout as-is for email communication. It can also be used for new incident types by adding the **Email Communication** layout to an incident type. See [Add the Email Communication Layout to an Incident Type](#add-the-email-communication-layout-to-an-incident-type) for details. 
```

Comment:
This will not work easily - the Email Communication pack depends on many Email related fields, for it to work on other incident types those fields must be associated with the new Incident Type

Reply:
Maybe we can add that It can also be used for new **email** related incident types